partially installed compiler

when I write opam switch create 5.1.0 (for example) and finish the process before completion, I still get 5.1.0 in the available versions of the compiler, although there are many errors when using it
I would like it to be visible when writing opam switch, which compiler is partially installed

I didn’t misunderstood this. I need to be able to see when you pressed Ctrl-C as I’m not able to reproduce your issue and timings are important for this kind of problem.

For example for me, interrupting it halfway leads to this output:
```
$ opam switch create 5.1.0

<><> Installing new switch packages <><><><><><><><><><><><><><><><><><><><>  🐫
Switch invariant: ["ocaml-base-compiler" {= "5.1.0"} | "ocaml-system" {= "5.1.0"}]

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫
∗ installed base-bigarray.base
∗ installed base-threads.base
∗ installed base-unix.base
∗ installed ocaml-options-vanilla.1
⬇ retrieved ocaml-base-compiler.5.1.0  (cached)
[ERROR] User interruption


<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫
┌─ The following actions failed
│ λ build ocaml-base-compiler 5.1.0
└─
┌─ The following changes have been performed (the rest was aborted)
│ ∗ install base-bigarray         base
│ ∗ install base-threads          base
│ ∗ install base-unix             base
│ ∗ install ocaml-options-vanilla 1
└─

<><> ocaml-base-compiler.5.1.0 troubleshooting ><><><><><><><><><><><><><><>  🐫
=> A failure in the middle of the build may be caused by build parallelism
      (enabled by default).
      Please file a bug report at https://github.com/ocaml/opam-repository/issues
=> You can try installing again including --jobs=1
      to force a sequential build instead.
Switch initialisation failed: clean up? ('n' will leave the switch partially installed) [y/n] y
```

--- a/master_changes.md
+++ b/master_changes.md
@@ -32,6 +32,7 @@ users)
 ## Remove
 
 ## Switch
+  * Precise message for Ctrl-c-ed opam switch creation "Switch left partially installed" [#5713 @rjbou - fix #5710]
 
 ## Config
 

--- a/src/client/opamSwitchCommand.ml
+++ b/src/client/opamSwitchCommand.ml
@@ -339,7 +339,11 @@ let create
     let () =
       try OpamStd.Exn.fatal e with e ->
         OpamStd.Exn.finalise e @@ fun () ->
-        OpamConsole.warning "Switch %s left partially installed"
+        OpamConsole.error
+          "Switch %s left %s installed.\n\
+           To clean it, use 'opam switch remove %s'"
+          (OpamConsole.colorise `underline (OpamSwitch.to_string st.switch))
+          (OpamConsole.colorise `bold "partially")
           (OpamSwitch.to_string st.switch)
     in
     OpamStd.Exn.finalise e @@ fun () ->

